src/pixelator/annotate/__init__.py

```diff
@@ -229,6 +232,15 @@ def annotate_components(
         json.dump(edgelist_metrics(edgelist), outfile, default=np_encoder)
 
 
+def _cluster_components_using_leiden(adata: AnnData) -> None:
```

Comment:
should we expose a `resolution` parameter here that defaults to 1.0? It is the only thing I can think of. But I think it works as is.

Reply:
Sure, I'll add that here.